mne/epochs.py

```diff
@@ -1529,7 +1530,7 @@ def as_data_frame(self, picks=None, index=None, scale_time=1e3,
         n_channel_types = 0
         ch_types_used = []
 
-        scalings = _mutable_defaults(('scalings', scalings))[0]
+        scalings = _mutable_default('scalings', scalings)
```

Comment:
We had tuples before for a reason ... this will change some intended behavrior somewhere I guess.